src/components/Settings/TwoStepFaq.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+	<div>
+		<kv-settings-card>
+			<template v-slot:content>
+				<h2>
+					Level up your security with 2-step verfication
+				</h2>
+				<p>
+					2-step verification protects your Kiva account by requiring an
+					additional code when you log in on a device we don't recognize.
+				</p>
+				<h3 class="strong">
+					FAQ
+				</h3>
+				<ul>
+					<li>
+						<kv-button
+							class="text-link"
+							@click="triggerWhatIsLightbox"
+						>
+							What is 2-step Verification?
+						</kv-button>
+					</li>
+					<li>
+						<kv-button
+							class="text-link"
+							@click="triggerHowItWorksLightbox"
+						>
+							How does it work?
+						</kv-button>
+					</li>
+					<li>
+						<kv-button
+							class="text-link"
+							@click="triggerHowItProtectsLightbox"
+						>
+							How it protects you?
+						</kv-button>
+					</li>
+				</ul>
+			</template>
+		</kv-settings-card>
+		<kv-lightbox
+			:visible="whatIsLbVisible"
+			@lightbox-closed="lightboxClosed"
+			title="What is 2-step verification?"
+		>
+			<p>
+				2-step verification is the most secure way to access your Kiva account.
+				By linking a second device to your account, you'll be required to enter
+				a code in addition to your password when you login.
+			</p>
+		</kv-lightbox>
+		<kv-lightbox
+			:visible="howItWorksLbVisible"
+			@lightbox-closed="lightboxClosed"
+			title="How does it work?"
+		>
+			<p>
+				Once you set up 2-step verification, you'll be asked to provide
+				a code from your mobile device in addition to your password in
+				order to log into your account. This code will be sent to you
+				via text, voice call, or a mobile app.
+			</p>
+		</kv-lightbox>
+		<kv-lightbox
+			:visible="howItProtectsLbVisible"
+			@lightbox-closed="lightboxClosed"
+			title="How it protects you"
+		>
+			<p>
+				2-step verification ensures that even if someone gains access to
+				your password, they won't be able to login without access to your
+				mobile device.
+			</p>
+		</kv-lightbox>
+	</div>
+</template>
+
+<script>
+import KvSettingsCard from '@/components/Kv/KvSettingsCard';
+import KvLightbox from '@/components/Kv/KvLightbox';
+
+export default {
+	components: {
+		KvSettingsCard,
+		KvLightbox,
+	},
+	data() {
+		return {
+			whatIsLbVisible: false,
+			howItWorksLbVisible: false,
+			howItProtectsLbVisible: false,
+		};
+	},
+	methods: {
+		triggerWhatIsLightbox() {
+			this.whatIsLbVisible = !this.whatIsLbVisible;
+		},
+		triggerHowItWorksLightbox() {
+			this.howItWorksLbVisible = !this.howItWorksLbVisible;
+		},
+		triggerHowItProtectsLightbox() {
+			this.howItProtectsLbVisible = !this.howItProtectsLbVisible;
+		},
+		lightboxClosed() {
+			this.whatIsLbVisible = false;
+			this.howItWorksLBVisible = false;
+			this.howItProtectsLBVisible = false;
+		},
+	}
+};
+</script>
+
+<style lang="scss" scoped>
+@import 'settings';
+
+ul {
```

Comment:
Minor thing, but when using scoped styles, there's a small performance hit if you use non-class selectors. 
From https://vue-loader.vuejs.org/guide/scoped-css.html 

> Scoped styles do not eliminate the need for classes. Due to the way browsers render various CSS selectors, p { color: red } will be many times slower when scoped (i.e. when combined with an attribute selector). If you use classes or ids instead, such as in .example { color: red }, then you virtually eliminate that performance hit.

So ideally this would be `.faq-list` or something instead of `ul`

Reply:
Same with the h2, h3